### ISSUE TITLE
SIGABRT When Video Ends Without Stacktrace

### ISSUE THREAD

**Issue by vindicatorr** [xbmc/xbmc#23301]
### Describe the bug
Here is a clear and concise description of what the problem is:

When a video ends or is stopped, kodi closes out with a SIGABRT crash.

## Expected Behavior
Here is a clear and concise description of what was expected to happen:

Not crash? Return back to the previous menu section (eg "TV/Recordings").

## Actual Behavior

Coredump

### To Reproduce

1.  Play Video
2. Stop Video (or wait until it ends on its own)
3. Reap crash reward.

### Debuglog

The debuglog can be found here:
[kodi_crashlog-20230518_150256.tar.gz](https://github.com/xbmc/xbmc/files/11511741/kodi_crashlog-20230518_150256.tar.gz)

### Screenshots 
Here are some links or screenshots to help explain the problem:

## Additional context or screenshots (if appropriate)
Here is some additional context or explanation that might help:

Reference: https://forum.kodi.tv/showthread.php?tid=373188

### Your Environment
Used Operating system:

 - [ ] Android
 - [ ] iOS
 - [ ] tvOS
 - [X] Linux
 - [ ] macOS
 - [ ] Windows
 - [ ] Windows UWP

 - Operating system version/name: Arch Linux
 - Kodi version: 21.0-ALPHA1 (20.90.101) Git:20230517-5f69284-dirty

*note: Once the issue is made we require you to update it with new information or Kodi versions should that be required.
Team Kodi will consider your problem report however, we will not make any promises the problem will be solved.*

**Comment by lrusak** [comment]
You'll need to post a stack trace otherwise it's impossible to know.

You'll likely need to install gdb and Kodi debug symbols to have the auto generated stack trace be useful.

**Comment by vindicatorr** [comment]
I had more details in my "reference" link above, which included the issue about not being able to get any stacktrace. In that link, I show how I even used `CMAKE_BUILD_TYPE` set to debug since I saw the cmake output without that set defaulted to release.

My title here even stated no stacktrace was provided.

**Comment by lrusak** [comment]
> I had more details in my "reference" link above, which included the issue about not being able to get any stacktrace. In that link, I show how I even used `CMAKE_BUILD_TYPE` set to debug since I saw the cmake output without that set defaulted to release.
> 
> 
> 
> My title here even stated no stacktrace was provided.

Do you have gdb installed like I said?

**Comment by vindicatorr** [comment]
I've had gdb installed for years. Do you want me to launch/attach to the process? Will it give me the stacktrace when it aborts?

**Comment by neo1973** [comment]
Could be a duplicate of #23285.

**Comment by vindicatorr** [comment]
Possible. These videos are also transport-streams (mpeg2), though played through tvheadend.

But note how that issue mentions it as a segmentation fault, while mine is showing as an abort.

I added a `video` folder that had an mp4 file, and that played through, and ended correctly by going back to the video folder.

But I sure would like to be able to get a stacktrace, and have yet been able to figure out why I'm not getting anything.

Another thought just came to me... I noticed another TV recording started, and that made me try viewing Live TV (because stopping videos, instead of just letting it end, also caused an abort).
I was able to stop Live TV, and remain alive in the EPG.
However, when I started the video for that TV channel currently being recorded, and stopped it manually, that DID cause it to abort.
